### PR TITLE
Advanced examples, lots of docs

### PR DESCRIPTION
**Pull request by Aylr**
## Things to look at

- [ ] advanced example file
- [ ] all the docs in the docs folder (and hopefully soon the readthedocs site)
- [ ] The new architecture overview doc for devs

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Welcome to healthcare.ai!
```
Comment (yvanhuele) [inline docs/index.md]: Change capitalization to be consistent with the other docs

File: docs/index.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 
-The aim of **healthcareai** is to streamline machine learning in healthcare. The package has two main goals:
+The aim of **healthcareai** is to streamline machine learning in healthcare with two main goals:
 
--  Allow one to easily create models based on tabular data, and deploy a best model that pushes predictions to SQL Server.
--  Provide tools related to data cleaning, manipulation, and imputation.
+- Allow one to easily create models based on tabular data, and deploy a good model that pushes predictions to a database (MSSQL, MySQL, SQLite, CSV).
+- Provide tools related to data cleaning, manipulation, and imputation.
 
 ## Where's the source code?
```
Comment (yvanhuele) [inline docs/index.md]: as above, change capitalization to be consistent with the other docs

File: docs/FAQ.md [hunk 3]
```diff
@@ -10,20 +10,20 @@ machine learning packages are certainly irreplaceable, we think that
...
 focuses on data issues specific to healthcare. This means that we pay
 attention to longitudinal questions, offer an easy way to do
 risk-adjusted comparisons, and provide easy connections and deployment
 to databases.
 
-## Who started this project?
+## Who Started This Project?
 
 This project began in the data science group at Health Catalyst, a Salt
 Lake City-based company based focused on improving healthcare outcomes.
```
Comment (Hanlei-Zhu) [inline docs/FAQ.md]: a Salt Lake City-based company focused on?

**Comment by yvanhuele** [comment]
I ran the advanced example and everything seemed to go fine.  One slightly confusing feature is that two of the curves in each of the ROC and PR graphs are labeled "RandomForestClassifier".  I assume this means the best performing model in the ensemble turned out to be a random forest, but wasn't sure so I figured I'd point it out just in case.

File: docs/FAQ.md (hunk 3 above)
Comment (Aylr) [inline docs/FAQ.md]: fixed

File: docs/index.md (hunk 2 above)
Comment (Aylr) [inline docs/index.md]: fixed

File: docs/index.md (hunk 1 above)
Comment (Aylr) [inline docs/index.md]: fixed

**Pull request merged**